scripts/pdf/render.py

```diff
@@ -67,14 +76,23 @@ def main(loc, colorscheme, output_filename):
             html += '<p style="page-break-before: always" ></p>'
             print(f"Rendered page {page_number} of the directory {operating_sys}")
 
+    output_filename_with_platform = output_filename
+    if platform:
+        output_filename_with_platform = (
+            f"{output_filename[:-4]}-{'+'.join(platform)}.pdf"
+        )
+
     html += "</body></html>"
 
     # Writing the PDF to disk
+    if not html.count("<h2"):
+        print(f"No pages found for platform {', '.join(platform)}. Skipping.")
```

Comment:
If someone hit this case where they pointed it an invalid directory or whatever without also specifying a platform, they'd hit a runtime exception here where `platform` is not defined.

Reply:
I guess I will remove this check then.